src/test/end-to-end/tests/verified/sjp-list-download-test.ts

```diff
@@ -0,0 +1,115 @@
+import { DateTime } from 'luxon';
+import { createLocation, uploadPublication } from '../../shared/testingSupportApi';
+
+Feature('SJP list download');
+
+Scenario('I as a verified user should be able to download sjp public list', async ({ I }) => {
+    const displayFrom = DateTime.now().toISO({ includeOffset: false });
+    const displayTo = DateTime.now().plus({ days: 1 }).toISO({ includeOffset: false });
+    const contentDate = DateTime.now().toFormat('dd MMMM yyyy');
+    const sjpListToDownload = 'Single Justice Procedure Public List ' + contentDate;
+    const locationId = '9';
+    const locationName = 'Single Justice Procedure';
+
+    await createLocation(locationId, locationName);
+    const artefactId = await uploadPublication(
+        'PUBLIC',
+        locationId,
+        displayFrom,
+        displayTo,
+        'ENGLISH',
+        'sjp-public-list.json',
+        'SJP_PUBLIC_LIST'
+    );
+
+    I.loginAsMediaUser();
+    I.waitForText('Your account');
+    I.see('Single Justice Procedure cases');
+    I.click('#card-summary-of-publications\\?locationId\\=9');
+    I.waitForText('What do you want to view from Single Justice Procedure?');
+    I.see(sjpListToDownload);
+    I.click(locate('//a').withText(sjpListToDownload));
+    I.waitForText('Single Justice Procedure cases that are ready for hearing');
+    I.see('Download a copy');
+    I.click('Download a copy');
+    I.waitForText('Terms and conditions');
+    I.see(
+        'As a verified user of the court and tribunal hearings service you are authorised to download this file containing personal protected data.'
+    );
+    I.see(
+        'It is your responsibility to ensure you comply with any GDPR and/or reporting restrictions regarding the content of this file.'
+    );
+    I.see('Please tick this box to agree to the above terms and conditions');
+    I.click('#disclaimer-agreement');
+    I.click('Continue');
+    I.waitForText('Download your file');
+    I.see('Save your file somewhere you can find it. You may need to print it or show it to someone later.');
+    I.see('If you have any questions, call 0300 303 0656.');
+
+    I.handleDownloads(artefactId + '.pdf');
```

Comment:
As discussed, we will need to decide what we want to do with the code below. As part of investigation for [PUB-2043](https://tools.hmcts.net/jira/browse/PUB-2043), we found out the the download is failing intermittently and so this test is going to cause build failure when run in staging.

Reply:
This is discussed.